package.json

```diff
@@ -1,136 +1,43 @@
 {
-  "name": "@qonto/ember-lottie",
+  "name": "ember-lottie",
```

Comment:
Should the name still be `@qonto/ember-lottie`?

Reply:
We should check on npm what's the good one, but I'm not sure if we actually need to keep the `@qonto`

Reply:
The name on npm is `@qonto/ember-lottie`: https://www.npmjs.com/package/@qonto/ember-lottie/v/0.5.0-alpha.0?activeTab=code

We are keeping `@qonto/ember-lottie` name but in the addon's package.json, it's the one that is published on npm. It was not possible to keep the same in the root package.json because pnpm was confused about it, that's why I out only `ember-lottie` in the root.